### DISCUSSION TITLE
How to write multiple commands for beforeBundleCommand?

### DISCUSSION THREAD

**Discussion by yk0n9** [tauri-apps/tauri#7266] [Q&A] [answered]
```build.beforeBundleCommand``` supports two parameters, one is ```string```, the other is ```object```, but I don’t know how to write ```object``` , I want to get some example

**Comment by FabianLars** [comment, marked as answer]
We're in the process of fixing the docs build pipeline so it can properly show these kind of config objects, but in this case the object looks like this:
```json
"beforeBundleCommand": {
  "script": "do something",
  "cwd": "/some/dir"
}
```
source: https://github.com/tauri-apps/tauri/blob/dev/core/tauri-config-schema/schema.json#L2113

So it doesn't help you to write multiple commands. You'll have to us `&&` or something to combine commands, or write a wrapper command in package.json for example.
